Tests/Unit/EmogrifierTest.php

```diff
@@ -967,27 +967,22 @@ public function emogrifyRemovesStyleNodes()
 
     /**
      * @test
+     *
+     * @expectedException \InvalidArgumentException
      */
-    public function emogrifyIgnoresInvalidCssSelector()
+    public function emogrifyFailsOnInvalidCssSelector()
     {
+        // HHVM ignore custom error handler settings for libxml
+        // @see https://github.com/facebook/hhvm/issues/5790
+        if (defined('HHVM_VERSION')) {
+            $this->markTestIncomplete('HHVM ignore custom error handler');
```

Comment:
This should be markTestSkipped, not markTestIncomplete. (markTestIncomplete serves as a kind of "to do" marker.)